app/views/ladder/components/Leaderboard.vue

```diff
@@ -431,4 +438,21 @@
       
     }
   }
+  #contact-us-info {
```

Comment:
Can we update the design for this section according to the mockup Brehan shared in Slack?

Reply:
sure